apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -1331,6 +1332,39 @@ async fn shutdown(settings: &Settings, project: &Project, graceful: GracefulShut
         let should_shutdown_containers = settings.should_shutdown_containers();
 
         if should_shutdown_containers {
+            super::display::show_message_wrapper(
```

Comment:
Why is this conditional on terminating the containers? Shouldn't we terminate anyway all the time?

Reply:
Good call, will update.